src/tensors/gpu/backend.h

```diff
@@ -29,9 +29,19 @@ class Backend : public marian::Backend {
 
   cublasHandle_t getCublasHandle() { return cublasHandle_; }
 
+#if CUDA_VERSION >= 9000
+  bool useTensorCores() {
+    return useTensorCores_;
+  }
+#endif
+
 private:
   cublasHandle_t cublasHandle_;
 
+#if CUDA_VERSION >= 9000
+  bool useTensorCores_{Config::useTensorCores};
```

Comment:
How about removing the #if, and instead turn this into a request? requestTensorCores_

Which will throw a runtime error if it cannot be fulfilled.